modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -140,6 +140,70 @@ The Taint-Based Evictions feature is enabled by default. The taints are automati
+    pods that tolerate the taint with a specified tolerationSeconds remain bound for the specified amount of time
+
+The node controller automatically taints a node when certain condition is true. The following taints are built in:
+
+    node.kubernetes.io/not-ready: Node is not ready. This corresponds to the NodeCondition Ready being “False”.
+    node.kubernetes.io/unreachable: Node is unreachable from the node controller. This corresponds to the NodeCondition Ready being “Unknown”.
+    node.kubernetes.io/out-of-disk: Node becomes out of disk.
+    node.kubernetes.io/memory-pressure: Node has memory pressure.
+    node.kubernetes.io/disk-pressure: Node has disk pressure.
+    node.kubernetes.io/network-unavailable: Node’s network is unavailable.
+    node.kubernetes.io/unschedulable: Node is unschedulable.
+    node.cloudprovider.kubernetes.io/uninitialized: When the kubelet is started with “external” cloud provider, this taint is set on a node to mark it as unusable. After a controller from the cloud-controller-manager initializes this node, the kubelet removes this taint.
+
+{product-title} has TaintBasedEvictions feature enabled by default, hence the taints are automatically added by the NodeController (or kubelet) and the normal logic for evicting pods from nodes based on the Ready NodeCondition is disabled.
+
+    Note: To maintain the existing rate limiting behavior of pod evictions due to node problems, the system actually adds the taints in a rate-limited way. This prevents massive pod evictions in scenarios such as the master becoming partitioned from the nodes.
+
+This feature, in combination with tolerationSeconds, allows a pod to specify how long it should stay bound to a node that has one or both of these problems.
+
+For example, an application with a lot of local state might want to stay bound to node for a long time in the event of network partition, in the hope that the partition will recover and thus the pod eviction can be avoided. The toleration the pod would use in that case would look like
+
+tolerations:
+- key: "node.kubernetes.io/unreachable"
+  operator: "Exists"
+  effect: "NoExecute"
+  tolerationSeconds: 6000
+
+Note that {product-title}  automatically adds a toleration for node.kubernetes.io/not-ready with tolerationSeconds=300 unless the pod configuration provided by the user already has a toleration for node.kubernetes.io/not-ready. Likewise it adds a toleration for node.kubernetes.io/unreachable with tolerationSeconds=300 unless the pod configuration provided by the user already has a toleration for node.kubernetes.io/unreachable.
+
+These automatically-added tolerations ensure that the default pod behavior of remaining bound for 5 minutes after one of these problems is detected is maintained. The two default tolerations are added by the DefaultTolerationSeconds admission controller.
+
+DaemonSet pods are created with NoExecute tolerations for the following taints with no tolerationSeconds:
```

Comment:
@openshift-ci-robot Is this still true? I am not seeing this; but, I am running an older version of 4.x. For example, the fluentd DS that Cluster Logging creates:
```
    tolerations:
        - key: node-role.kubernetes.io/master
          operator: Exists
          effect: NoSchedule
      priorityClassName: cluster-logging
```